src/utils/index.ts

```diff
@@ -295,6 +294,41 @@ export const fetchTranscriptDetails = (allTranscripts: Transcript[], paths: stri
   };
 };
 
+export const createLanguageTreeArray = (languageTree: any) => {
```

Comment:
lets have the type here instead of any.
It has been defined in `https://github.com/bitcointranscripts/registry/pull/62/files#diff-b16cf354102592c3c6fcdb23db6b2be79541567bae9e23471dd74a59f1a35a2cR46`